src/plugins/intel_gna/tests/functional/limitations/layers_limit.cpp

```diff
@@ -9,7 +9,7 @@
 #include "shared_test_classes/base/layer_test_utils.hpp"
 
 using namespace InferenceEngine;
-// using namespace ov::test;
+using namespace ov::opset8;
```

Comment:
opset9?

Reply:
This is merged fix, the same test in master